openprescribing/templates/entity_home_page.html

```diff
@@ -201,8 +201,9 @@ <h3>Looking for more data?</h3>
           <a href="{{ chartTitleUrl }}">{{ chartTitle }}</a>
         </span>
       </div>
-      <div class="panel-body" id="{{ chartId }}" data-costsaving="{{ cost_saving }}">
-        <div class="chart">
```

Comment:
There's now no longer a div with class `chart`. Does this matter?

Reply:
I'm pretty sure it doesn't matter.  There's been a lot of copying/pasting/fiddline with the chart template code, and so there are number of unnecessary tags in there.

Further rationalisation of chart template code is possible.